src/routes/paths.ts

```diff
@@ -40,6 +40,7 @@ export const paths = {
   epoch: '/epoch',
   give: '/give',
   vouching: '/vouching',
+  devPortal: '/devPortal',
```

Comment:
I prefer case insensitive URLs, so maybe `/devs`,  `/developers` or  `/api` as URLs to use instead?